app/celery/broadcast_message_tasks.py

```diff
@@ -1,17 +1,54 @@
 import uuid
+from datetime import datetime
 
 from flask import current_app
 from notifications_utils.statsd_decorators import statsd
 from sqlalchemy.schema import Sequence
+from celery.exceptions import MaxRetriesExceededError
 
 from app import cbc_proxy_client, db, notify_celery
+from app.clients.cbc_proxy import CBCProxyException
 from app.config import QueueNames
 from app.models import BroadcastEventMessageType, BroadcastProvider
 from app.dao.broadcast_message_dao import dao_get_broadcast_event_by_id, create_broadcast_provider_message
 
 from app.utils import format_sequential_number
 
 
+def get_retry_duration(retry_count):
```

Comment:
Optional: Rename `get_retry_delay` as `delay` is the wording used by celery, rather than `duration` for this - https://docs.celeryproject.org/en/3.1/userguide/tasks.html#using-a-custom-retry-delay